src/components/Form/MucInput.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div
+    class="m-form-group"
+    :class="isErrorClass"
+  >
+    <label
+      for="search-input"
+      class="m-label"
+    >
+      {{ label }}
+      <span
+        v-if="required"
+        aria-hidden="true"
+        class="mandatory"
+      >
+        *
+        <span class="visually-hidden">(erforderlich)</span>
+      </span>
+    </label>
+    <p
+      id="text-input-error"
+      class="m-error-message"
+    >
+      {{ errorMsg }}
+    </p>
```

Comment:
Should we hide this if no errorMsg is set?

Reply:
Thought about it, but it does not take up any space on page ... one less v-if?

Reply:
Then we can leave it as is 👍